Ready to be reviewed

Tavisca-vvinod opened the pull request.

Tavisca-vvinod committed:
Solev the problem

Tavisca-vvinod committed:
Update Program.cs

Tavisca-vvinod committed:
Used DateTime and revised the code.

arpit-nagar commented:
This code only works for mentioned scenario. 
Logic is hard coded as scenario. Need to update the logic.
Lets have a telephonic conversation to understand the problem statement first and then you need to fix the logic.

Tavisca-vvinod committed:
Updated Program.

@arpit-nagar, sir please check this one, I have tried to get a deeper understanding of the problem.

Tavisca-vvinod committed:
Updated Program with a more deeper understanding.

@arpit-nagar , sir please review.

Tavisca-vvinod committed:
Updated program with complete understanding.

@arpit-nagar Sir, this is my final code. Please review this.
